controllers/useraccount/console_usersettings_test.go

```diff
@@ -0,0 +1,285 @@
+package useraccount
+
+import (
+	"context"
+	"fmt"
+	"github.com/codeready-toolchain/toolchain-common/pkg/test"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	rbac "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"testing"
+)
+
+func TestDeleteConsoleSettingObjects(t *testing.T) {
+	t.Run("Delete ConfigMap", func(t *testing.T) {
+		t.Run("Configmap found by name and deleted", func(t *testing.T) {
+			ctx := context.Background()
+			cm := &corev1.ConfigMap{
+				TypeMeta: metav1.TypeMeta{Kind: "ConfigMap"},
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "user-settings-johnsmith",
+					Namespace: UserSettingNS,
+				},
+			}
+			cl := fake.NewClientBuilder().WithObjects(cm).Build()
```

Comment:
is there any reason for not using our own FakeClient?

Reply:
no, missed this one - fixing it now

Reply:
fixed in [7b7bd25](https://github.com/codeready-toolchain/member-operator/pull/568/commits/7b7bd259bb5cb92d555a5043585a0d3c06875e8b)